ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -485,4 +487,42 @@ Get Workbench Pod
     ${_}  ${cr_name}=    Get Openshift Notebook CR From Workbench
     ...    workbench_title=${workbench_title}  namespace=${ns_name}
     ${pod_info}=    Oc Get    kind=Pod  name=${cr_name}-0  api_version=v1  namespace=${ns_name}
-    RETURN    ${ns_name}    ${cr_name}    ${pod_info}
\ No newline at end of file
+    RETURN    ${ns_name}    ${cr_name}    ${pod_info}
+
+Edit Workbench
```

Comment:
one critical point for this kw and the Create Workbench one will be maintenance: every time we need to change the logic for one field, we need to change in 2 places. It's partially already split into lower level keywords like  `Add Environment Variables In Workbench`, but many checks still remain duplicated. 

Would mind trying refactor the Create Workbench keyword so that we reduce everything to single kw calls? This would make easier to get a well defined `Edit Workbench` kw.

Or alternative, we could have many "Edit" keywords, like the one already present for GPUs: `Edit GPU Number`